collections/sentinel-s2-l2a-mosaic-120/README.MD

```diff
@@ -1,23 +1,46 @@
-# Sentinel-2 L2A 120m Mosaic 
+# Sentinel-2 L2A 120m Mosaic
 
 ## Short description
 
-Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As there are some parts of the world, which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available [here](https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/).    
+Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As clouds can be missed and as there are some parts of the world which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available [here](https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/).
 
-## Band information  
-Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-2-l2a/#available-bands-and-data)
+## Band information
+Sentinel-2 L2A 120m mosaic contains 12 Sentinel 2 L2A bands and 1 quality mask. Find more information about sentinel 2 L2A bands [here](https://docs.sentinel-hub.com/api/latest/data/sentinel-1-l2a/#available-bands-and-data).
```

Comment:
no idea how we now got a wrong url name  `sentinel -1-l2a `

revert back to sentinel -2-l2a

Reply:
Good catch, no idea how this happened.

Reply:
Fixed.